### DISCUSSION TITLE
Any way to alter a response to return extra (programmatically) calculated data?

### DISCUSSION THREAD

**Discussion by starkmarkler** [usebruno/bruno#2451] [Q&A]
Hi! 

I work with a ton of time series data.  I'd love to convert all my epoch responses into a human readable date.  Is it possible to alter (append) the response to do this?  I recognize that it would have to be programmatic, and that's totally fine.

Thanks,

Ryan

**Comment by Its-treason** [comment]
Its not documented, but you can overwrite `res.res.data` like this:
```js
res.res.data = '{"overwridden": "body"}';
res.res.headers['content-type'] = 'application/json';
```
Overwritting the content-type header will ensure the response body is correctly formatted.

**Reply by gajanak** [reply]
It Seems this is not working in v1.39.1 anymore. 
Error: Error: cannot read property 'headers' of undefined.
